src/examples/ExpanderExamplePage.tsx

```diff
@@ -72,7 +72,9 @@ export const ExpanderExamplePage: React.FunctionComponent<{}> = () => {
       ]}>
       <Example title="A simple Expander." code={example1jsx}>
         <Expander collapsedHeight={50} expandedHeight={125}>
-          <Text>Text in the header</Text>
+          <Text accessible accessibilityRole="button">
```

Comment:
was setting accessibilityRole here intentional?

Reply:
Actually after looking at it again I don't think we need an accessibilityRole here since it's not the actual heading of the page